.codeclimate.yml

```diff
@@ -38,3 +38,4 @@ exclude_paths:
 - app/api/search_consumer/
 -  "**/vendor/**/*"
 - app/assets/images/
+- scripts/
```

Comment:
Note: I realized that any failures in that directory are probably related to my original reporting code. I'll run take a look at the failures....

Reply:
Hey Martha, I added the scripts directory and a skip over it here because the coverage script was being considered as part of the total coverage. I tried adding it in another path under lib>tasks>check_coverage and tried to specifically ignore that path only but that seemingly didn't work, it seemed to be related to a config issue I couldn't figure out. The other alternative would be to save it under the same path but write a test for it to account for it's coverage. Would that be a better option?

Reply:
Ah, I see. In that case, you could skip the check for that specific file, rather than the whole directory.